Add api to show hide series without removing it #471 (#581)
tradingview/lightweight-charts: docs/series-basics.md

});
```

### Visibility

When adding any series to a chart, you can specify its visibility using `visible` property.

> By default, created series is visible. You can change its visibility using `SeriesApi.applyOptions`:

Reviewer: ```suggestion
By default, all created series are visible. You can change the visibility using `applyOptions` method:
```

```js
series.applyOptions({
    visible: false,
});
```

> When you hide a series - price line, base line, price labels and markers of series will also be hidden.

Reviewer: ```suggestion
If the series is hidden - everything including its price lines, base line, price labels and markers will be hidden as well.
```

> Note that hiding a series is not equivalent to deleting it, you can show it again:
> 
> ```js
> series.applyOptions({
>     visible: true,
> });
> ```

Reviewer: ```suggestion
Note that hiding the series is not equivalent to deleting it - hiding does not affect the time scale at all in opposite of deleting, when the time scale might be changed (some points might be deleted).
```

|Name|Type|Default|Description|
|----|----|-------|-|

> |`priceLineVisible`|`boolean`|`true`|If true, a series' price line is displayed on a chart if a series is visible|

Reviewer 2: ```suggestion
|`priceLineVisible`|`boolean`|`true`|If true, a series' price line is displayed on a chart if the series is visible|
```

|Name|Type|Default|Description|
|----|----|-------|-|

> |`lastValueVisible`|`boolean`|`true`|If true, the label with the current price is displayed on the price scale if a series is visible|

Reviewer 2: ```suggestion
|`lastValueVisible`|`boolean`|`true`|If true, the label with the current price is displayed on the price scale if the series is visible|
```

|Name|Type|Default|Description|
|----|----|-------|-|

> |`baseLineVisible`|`boolean`|`true`|If true, a series' base line is displayed on a chart if a series is visible|

Reviewer 2: ```suggestion
|`baseLineVisible`|`boolean`|`true`|If true, a series' base line is displayed on a chart if the series is visible|
```
